qemu/tests/rng_bat.py

```diff
@@ -120,17 +118,22 @@ def get_driver_id(session, cmd, pattern):
         logging.info("register 'viorngum.dll' into system")
         session.cmd(rng_dll_register_cmd, timeout=120)
 
+    start_rngd_service = params.get("start_rngd_service")
+    if start_rngd_service:
```

Comment:
I prefer to check if the service is running and only start it when it is not.

Reply:
Updated, thanks!